lib/checks/keyboard/focus.js

```diff
@@ -0,0 +1,7 @@
+if (node.getAttribute('onFocus') === 'this.blur()') {
+  return false;
+}
+if (node.getAttribute('onFocus').indexOf('this.blur()') > -1) {
```

Comment:
Sorry, this is a little nit-picky, but HTML attributes are all lower case. We should stick to that convention. In this particular case it doesn't matter, but there are scenarios where upper case in attributes can cause problems, so as a general practice we should just make sure our use of things match what is defined in the spec.